ArduPlane/servos.cpp

```diff
@@ -470,16 +470,6 @@ void Plane::set_servos_controlled(void)
         SRV_Channels::set_output_scaled(SRV_Channel::k_throttle, 
             constrain_int16(quadplane.forward_throttle_pct(), min_throttle, max_throttle));
     }
-
-#if SOARING_ENABLED == ENABLED
-    // suppress throttle when soaring is active
```

Comment:
did you mean to remove this?

Reply:
Yes. This is now handled by setting a gliding_requested flag in the TECS, which will then output zero throttle. So this override is no longer required